UPGRADE.md

```diff
@@ -170,6 +170,8 @@ You can bring back previous configuration by overriding current routing with you
   * `findCartForSelectingShipping($id): ?OrderInterface`
   * `findCartForSelectingPayment($id): ?OrderInterface`
 
+* `Channel` relation was removed from `ChannelPricing` model. `Channel::$code` should be used instead.
```

Comment:
I meant, that instead of relation to `Channel` on `ChannelPricing`, the `Channel` `$code` property is used